examples/material_gallery/lib/demo/leave_behind_demo.dart

```diff
@@ -67,6 +67,19 @@ class LeaveBehindDemoState extends State<LeaveBehindDemo> {
     }
   }
 
+  void handleUndo(LeaveBehindItem item) {
+    int insertionIndex = leaveBehindItems.length;
```

Comment:
I used the lowerBound() function which should be log(N). It's from `package:collection/collection.dart`, not `dart:collection`.